testdata/templates/args.yaml

```diff
@@ -0,0 +1,8 @@
+service: foo
+headers:
+  header1: ${user:foo}
```

Comment:
Yep, we want to use it to load into the `Template` struct (known Go shape). Won't affect the body which is still a map.